guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collections.adoc

```diff
@@ -0,0 +1,124 @@
+[id="creating-a-playbook-with-modules-from-{project-context}-ansible-collections"]
+= Creating a playbook with modules from {Project} Ansible Collections
+
+Ansible modules from the {Project} Ansible Collections must be able to communicate with the {Project} API over HTTPS.
+In your playbooks, include parameters specifying how to authenticate to enable the connection to API.
+
+[IMPORTANT]
+====
+Do not store sensitive credentials, such as username and password, directly in your playbooks or environment variables.
+The following examples use Ansible vault to manage sensitive data.
+====
+
+.Prerequisites
+* A user account in {Project} exists with permissions to perform the action defined in your playbook.
+* You have access to one of the following types of authentication credentials for that user:
+** Username and password
+** Kerberos ticket, if your {ProjectServer} is configured to use {FreeIPA} as an external authentication source
+** Username and {Project} Personal Access Token
+* A system that can reach the {Project} API.
+This is the system that you will execute your playbook against.
+It can be {ProjectServer} itself.
+The following examples use `localhost`.
+
+.Procedure
+. Store your sensitive variables in an encrypted file:
+.. Create an Ansible vault.
+For example, to create a vault named _My_Vault.yml_:
++
+[subs="+quotes,attributes"]
+----
+$ ansible-vault create _My_Vault.yml_
+----
+.. After the `ansible-vault create` command opens an editor, provide the required parameters in the _key_: _value_ format.
++
+If you want to authenticate with {Project} user credentials:
++
+[source,ini,subs="+quotes,attributes"]
+----
+_My_Server_URL_: _https://{foreman-example-com}_
+_My_Username_: _My_Admin_User_Account_
+_My_Password_: _My_Admin_Password_
+----
++
+If you want to authenticate with a Kerberos ticket or a Personal Access Token:
++
+[source,ini,subs="+quotes,attributes"]
+----
+_My_Server_URL_: _https://{foreman-example-com}_
+----
+.. Save the changes, and close the editor.
+Ansible encrypts the data in the vault.
+. Create a playbook file that references the vault file:
++
+[NOTE]
+====
+The following YAML snippets include the `module_defaults` keyword to define vault parameters for all modules from the {ansible-module-defaults-group} group that are used in the playbook.
+Module defaults groups simplify parameter management by enabling you define common parameters to groups of modules rather than having to pass the parameters to each module individually.
```

Comment:
```suggestion
The following YAML snippets include the `module_defaults` keyword to pass vault variables as parameters for all modules from the {ansible-module-defaults-group} group that are used in the playbook.
Module defaults groups simplify parameter management by enabling you define common parameters to groups of modules rather than having to pass the parameters to each module individually.
```

Can/should we link https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_module_defaults.html#module-defaults-groups? It's not very extensive, but gives a bit more background at least?